rust/worker/src/system/scheduler.rs

```diff
@@ -115,6 +112,7 @@ impl Scheduler {
         self.handles.write().push(handle);
     }
 
+    #[cfg(test)]
```

Comment:
I don't think this is meant to be test-only exposed, its library code that should be available to use

Reply:
It's only used in tests.  We can make it public if you'd prefer.